p2p/simulations/http.go

```diff
@@ -465,6 +465,7 @@ func (s *Server) StreamNetworkEvents(w http.ResponseWriter, req *http.Request) {
 	}
 
 	w.Header().Set("Content-Type", "text/event-stream; charset=utf-8")
+	fmt.Fprintf(w, "\n\n")
```

Comment:
This should be after `w.WriteHeader` (otherwise we will try to write the header twice).